app/src/main/res/values/strings.xml

```diff
@@ -117,24 +117,27 @@
         like better. With enough comparison picks, the app will create a ranking from whom you like
         the most to least.\n
         \n
-        Next the app limits every user to maximum 10 matches. You get matched based on whom you
+        The app limits every user to maximum 10 matches. You get matched based on whom you
         like the most and who likes you back. If you both are super popular, chances are you\'ll get
-        matched up unless either of you runs out of matches.\n
+        matched up unless either of you runs out of matches. If you are more "average" attractive, chances 
+        are that your top choices will run out of matches before you\'d get considered. 
+        You\'ll likely match with other average attractive people who also couldn\'t get their first choices.
         \n
+       
         Of course, there is a chance that you and your main squeeze are a niche. Say you are both
-        into punk. You might like each other back as number one while the rest of the dating world couldn't
-        care less about you two.
+        into punk. You might like each other back immediately and hit it off.
     </string>
     <string name="about_background_heading">Background</string>
     <string name="about_background_body_text">
-        We've all been there: messages that go nowhere; likes, winks, and pokes sent with no response; 
-        conversations that fizzle and die. How can it be so hard to meet someone you like and who actually likes you back?\n
+        We've all been there: hours of swiping that go nowhere; annoyingly creepy messages;
+        conversations that fizzle and die. It can be damn hard to meet someone you like who also likes you back.\n
         \n
-        Fortunately, Beeline's got your back. It slots you with only a small number of matches that are 
-        genuinely interested in you.\n
+        Fortunately, Beeline's got your back. Through your comparisons, it knows exactly how much you are attracted to each person 
+        and who is highly attracted back. Beeline then matches you ONLY with 
+        people who have the highest mutual attraction. Think mutual control, likes, and potential. \n 
```

Comment:
This is all very strong and excels at condensing a lot with a conversational and fun tone.

"Think mutual control, likes, and potential."
This sentence is misleading. The app should be distanced from "likes." People hand those out casually. The app also doesn't calculate a potential OKCupid and eHarmony advertise algorithms based on statistics and psychology research to predict potential. Beeline doesn't put stock in those algorithm predictions but trusts the gut instinct/reaction of the users.